src/utils.py

```diff
@@ -69,3 +85,35 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def semver_labels_exist_in_pull_labels(pull):
+    return any([label for label in get_labels(pull=pull) if label in SEMVER_LABELS_SET])
+
+
+def get_semver_user_input(body):
+    match = re.match(
+        rf".*\B/(?P<{SEMVER}>{LABEL_KEY_MAJOR}|{LABEL_KEY_MINOR}|{LABEL_KEY_PATCH})\b",
+        body,
+        re.IGNORECASE,
+    )
+    if match:
+        return match.group(SEMVER).lower()
+
+
+def get_semver_label_to_add_from_user_input(body):
+    semver_user_input = get_semver_user_input(body=body)
+    return SEMVER_USER_INPUT_LABELS_MAP.get(semver_user_input)
```

Comment:
why do we need to use `get` here? the list should be a predefined one and semver_user_input will be None if the regex is not matched.